packages/python/flojoy_cloud/client.py

```diff
@@ -387,3 +388,49 @@ def get_measurement(self, measurement_id: str):
     @query(model=None)
     def delete_measurement(self, measurement_id: str):
         return self.client.delete(f"/measurements/{measurement_id}")
+
+    @query(model=None)
+    def get_hardware_id(self, part_number: str, serial_number: str):
+        return "TODO"
+
+    @query(model=MeasurementCreateResult)
+    def upload_session(
+        self,
+        serial_number: str,
+        station_id: str,
+        integrity: bool,
+        aborted: bool,
+        notes: Optional[str],
+        commit_hash: Optional[str],
+        measurements: List[SessionMeasurement],
+    ):
+        m_list = []
+        for m in measurements:
+            # Find id of each part
+            m_list.append(
+                {
+                    "data": make_payload(m.data),
+                    "name": m.name,
+                    "pass": m.passed,
+                    "createdAt": m.created_at.isoformat() if m.created_at else None,
+                }
+            )
```

Comment:
why is testId removed?

Reply:
I'll revert the changed for `flojoy_cloud` pkgs. That was for testing purposed.
I'm did not bother with the test id, even in the sequencer. I'm adding it back later today